pkgs/development/python-modules/amaranth/default.nix

```
@@ -57,6 +57,8 @@ buildPythonPackage rec {
     substituteInPlace setup.py \
       --replace "Jinja2~=2.11" "Jinja2>=2.11" \
       --replace "pyvcd~=0.2.2" "pyvcd"
+    substituteInPlace amaranth/build/plat.py \
```

do you mind adding a comment for future context?
```suggestion
   # jinja2.contextfunction was removed in jinja2 v3.1.
    substituteInPlace amaranth/build/plat.py \
```

Also, if we could upstream this patch, that would be great.

---

done!